aiida_vasp/assistant/parameters.py

```diff
@@ -187,38 +176,40 @@ def _load_valid_params(self):
         # Now add any unsupported parameter to the list
         for key, _ in self._unsupported_parameters.items():
             key = key.lower()
-            try:
-                _ = self._massage[key]
-                raise ValueError(f'The supplied unsupported_parameters with key {key} is already a supported parameter.')
-            except KeyError:
+            if key not in self._valid_parameters:
                 self._valid_parameters.append(key)
 
     def _set_parameters(self):
         """Iterate over the valid parameters and call the set function associated with that parameter."""
         for key in self._valid_parameters:
             self._set(key)
-            # We check after each parameter set if there is an exit code set on the WorkChain, if so, return
-            if self.exit_code is not None:
-                return
 
     def _set_override_parameters(self):
         """Set the any supplied override parameters."""
         try:
             if self._parameters.code:
-                self._massage.code = AttributeDict()
-            for key, item in self._parameters.code.items():
-                # Sweep the override input parameters to check if they are valid VASP tags
-                key = key.lower()
-                if self._valid_vasp_parameter(key):
-                    self._massage.code[key] = item
-                else:
-                    break
+                for key, item in self._parameters.code.items():
```

Comment:
For backward compatibility:

Here, we can search for keys in both `code` and `vasp` namespace and combine them into self._message.code

Reply:
We can for instance do this and issue a deprecation warning. Eventually I think it is for the better to come up with something else. It should be something that represents `incar`, but is not called that of course. See other reply.